Support disabling placeholders

nmielnik opened the pull request:
When `disablePlaceholders` is passed as an option to medium-editor:
- Do not add/remove the 'medium-editor-placeholder' class on the contenteditable element
- Don't attach event handlers which had the sole purpose of handling when a placeholder should be shown/hidden

daviferreira commented:
Hey @nmielnik, thanks for all your work. Could you be so kind to update the README with your new options? Cheers!

nmielnik commented:
Thanks for pulling this in!
I've updated the documentation (#357) to cover everything that @nchase and I had added, as well as the 'staticToolbar' and 'stickyToolbar' options, when I noticed weren't mentioned in the documentation.
